back-end/controllers/record-controller.js

```diff
@@ -42,30 +44,32 @@ const recordController = {
 
       // Check if there are missing data
       if (!title || !amount || !categoryId)
-        return res.json(
-          'missing title or amount or category id to update this record'
-        )
+        return res
+          .status(400)
+          .json('missing title or amount or category id to update this record')
 
       // Check if the record is in database
       const theRecord = await prisma.record.findUnique({
         where: {
           id: recordId,
         },
       })
-      theRecord ? theRecord : res.json('the record does not exist')
+      theRecord ? theRecord : res.status(400).json('the record does not exist')
```

Comment:
I assume that you want to check if `theRecord` exists (not `undefined`) or return status code `400`.

If that's the case, I'd suggest rewriting like this:

```js
if (!theRecord) res.status(400).json('the record does not exist')
```

It's clear that you're checking the result of the prisma query.